README.md

```diff
@@ -1 +1,61 @@
-# Module1-Hello-Microverse
+# Project Name
+Hello microverseinc
+
+## Built With
+- Html 
+- Css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ebukanweje.github.io/Module1-Hello-Microverse/)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/EbukaNweje)
+- Twitter: [@twitterhandle](https://twitter.com/NwejeEbuka)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ebuka-nweje-b5993317a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] [OPTIONAL] Kindly add the MIT.md file to you repository, because now this link is broken 👍